coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "line": 92.35
+    "line": 97.2
```

Comment:
you can also ignore the coverage directory so that you don't have to keep committing coverage report files